server/database/seed.js

```diff
@@ -5,20 +5,35 @@ const constants = require('./constants');
 
 faker.seed(86753092);
 
+const tracker = (name, target, next, args) => {
+  let counter = 0;
+  return (error, results) => {
+    if (error) {
+      console.log(`Error while executing query inside ${name}:`, error);
+      return;
+    }
+    counter += 1;
+    console.log(`Query executed ${counter} times inside ${name}`);
```

Comment:
Is this console.log necessary for production? The way I understand the script, this will fire after every db query. It seems like it would really clutter up your console when generating 10M+ records.

Reply:
Yes, it should not be part of the production script. I used it to make sure everything works as expected.